components/schemes/Typography/tests/unit/MDCTypographySchemeTests.m

```diff
@@ -68,4 +71,48 @@ - (void)testTypographySchemeCopy {
   XCTAssertEqualObjects(typographyScheme.overline, typographySchemeCopy.overline);
 }
 
+// In order to avoid unintended consequences for our clients, ensure that our earlier typography
+// scheme does NOT return fonts with associated scaling curves.
+- (void)testTypographyScheme201804DoesNotHaveCurves {
+  // Given
+  MDCTypographyScheme *scheme201804 =
+      [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
+
+  // Then
+  XCTAssertNil(scheme201804.headline1.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.headline2.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.headline3.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.headline4.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.headline5.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.headline6.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.subtitle1.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.subtitle2.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.body1.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.body2.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.caption.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.button.mdc_scalingCurve);
+  XCTAssertNil(scheme201804.overline.mdc_scalingCurve);
+}
+
+- (void)testTypographyScheme201902HaveCurves {
+  // Given
+  MDCTypographyScheme *scheme201902 =
+      [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201902];
+
+  // Then
+  XCTAssertNotNil(scheme201902.headline1.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.headline2.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.headline3.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.headline4.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.headline5.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.headline6.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.subtitle1.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.subtitle2.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.body1.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.body2.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.caption.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.button.mdc_scalingCurve);
+  XCTAssertNotNil(scheme201902.overline.mdc_scalingCurve);
+}
```

Comment:
Please add a snapshot test to capture the new fonts.

Reply:
Add a tracking issue #6959